marketplace/core/types/channels/weni_web_chat/serializers.py

```diff
@@ -13,3 +23,102 @@ def create(self, validated_data):
 
         validated_data["platform"] = WeniWebChatType.platform
         return super().create(validated_data)
+
+
+class AvatarBase64ImageField(Base64ImageField):
+    def get_file_name(self, extencion: str) -> str:
+        return f"avatar.{extencion}"
+
+
+class ConfigSerializer(serializers.Serializer):
+    title = serializers.CharField(required=True)
+    subtitle = serializers.CharField(required=False)
+    inputTextFieldHint = serializers.CharField(default="Type a message...")
```

Comment:
are you using the camel case because of the according with @paulobernardoaf right? Is there a way to use a translation for snake and camel case?

Reply:
Yes, both for the data that I will receive and to assemble the script I need these fields in the standard camel case.